builtin/logical/pki/crl_util.go

```diff
@@ -38,8 +58,9 @@ type revocationInfo struct {
 //
 // The CRL builder also tracks the revocation configuration.
 type crlBuilder struct {
-	m            sync.Mutex
-	forceRebuild uint32
+	m                     sync.Mutex
```

Comment:
I didn't mind the one lock inside crlBuilder, but it would be really great if we could have a better name here?  I don't know what "m" stands for, so I ended up going back to check if this was config or soemthing else.